utils/test-utils/src/lib.rs

```diff
@@ -1,6 +1,20 @@
 use frame_system::Config;
 use pretty_assertions::assert_eq;
 
+/// Check if an event of type `TEvent` is present anywhere in system events.
+pub fn expect_event<TEvent: std::fmt::Debug + PartialEq, TRuntime: Config>(e: TEvent)
```

Comment:
Nice, thanks. I was always implementing it in my tests. But I would suggest to use all events using `frame_system::Pallet::<TRuntime>::events()` to avoid using some constant numbers.